config/initializers/i18n_defaults.rb

```diff
@@ -0,0 +1,10 @@
+i18n_simple_backend = I18n::Backend::Simple.new
+old_handler = I18n.exception_handler
+I18n.exception_handler = lambda do |exception, locale, key, options|
+  case exception
+  when I18n::MissingTranslation
```

Comment:
I think this might be totally uncaught if the `key` is missing from `:en` which is bad.